src/com/nick/ttt/TicTacToeGame.java

```diff
@@ -47,20 +69,45 @@ public void drawBoard() { // visual display for the human
     }
 
     public void placePiece(char xo, int pos) {
-        this.getGameBoard()[pos] = xo;
+        if (this.findEmpties().contains(pos)) {
```

Comment:
The check that verifies a valid move should be separate from the method that places it. Place Piece can make a call to the verifier. But the verifier should be distinct (we will use it for our AI.)